tools/tsp-client/.gitignore

```diff
@@ -3,3 +3,5 @@ node_modules
 dist
 types
 temp
+tsp-output/
+output/
```

Comment:
Just curious where does output/ come from?

Reply:
The tool places the output files in `output` while `tsp-output` is where `tsp compile` will generate to.